tests/test_modules.py

```diff
@@ -10,7 +10,8 @@
 from buildtest.tools.modulesystem.module_difference import diff_trees
 from buildtest.tools.modulesystem.tree import module_tree_add, module_tree_rm
 from buildtest.tools.log import BuildTestError
-from buildtest.module import Module, ModuleCollection,get_all_collections
+from buildtest.module import Module, ModuleCollection, get_all_collections
+
```

Comment:
naming question - is there some reason we would want to name the function `get_all_collections` as opposed to just `get_collections`? My expectation would be that `get_collections` would return all by default, but perhaps there is some variable I could provide to filter, etc. If we name `get_all_collections` and then wanted to add such a variable, it would be weird.